From 908208859db833306fdb37c13d8fa525ad4b6696 Mon Sep 17 00:00:00 2001
From: "Panwar, Ashish" <ashish.panwar@intel.com>
Date: Fri, 11 Mar 2016 06:00:19 +0530
Subject: [PATCH 1899/2508] ASoC: Intel: Boards: Fix stream name for dai_link.

commit 0590828d4986d89ee1d283ebfb928881d8212db1 from
https://github.com/01org/linux-apollolake-i

Use a clean name for dai_link as it is being exposed to userspace
via compress node proc interface. It should help user to figure
out the device objective.

Change-Id: Ic45dd529e1f9e20ef7ad454e253bac25de747cf8
Signed-off-by: Panwar, Ashish <ashish.panwar@intel.com>
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/8375
Reviewed-by: Babu, Ramesh <ramesh.babu@intel.com>
Tested-by: Babu, Ramesh <ramesh.babu@intel.com>
---
 sound/soc/intel/boards/bxtp_ivi_rse_rt298.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/intel/boards/bxtp_ivi_rse_rt298.c b/sound/soc/intel/boards/bxtp_ivi_rse_rt298.c
index 3518e50..445c87c 100644
--- a/sound/soc/intel/boards/bxtp_ivi_rse_rt298.c
+++ b/sound/soc/intel/boards/bxtp_ivi_rse_rt298.c
@@ -263,7 +263,7 @@ static struct snd_soc_dai_link broxton_rt298_dais[] = {
 	/* Trace Buffer DAI links */
 	{
 		.name = "Bxt Trace Buffer0",
-		.stream_name = "dsp0tracebuffer",
+		.stream_name = "Core 0 Trace Buffer",
 		.cpu_dai_name = "TraceBuffer0 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -273,7 +273,7 @@ static struct snd_soc_dai_link broxton_rt298_dais[] = {
 	},
 	{
 		.name = "Bxt Trace Buffer1",
-		.stream_name = "dsp1tracebuffer",
+		.stream_name = "Core 1 Trace Buffer",
 		.cpu_dai_name = "TraceBuffer1 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
-- 
1.7.5.4

